### PR TITLE
added to clock docs which adds to the issue #2690

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
The PR adds the themes to the documentation for clock 
Also adds an example for a custom clock in storybook
#### Where should the reviewer start?
Clock/doc.js
#### What are the relevant issues?
#2690
#### Do the grommet docs need to be updated?
auto
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

File: src/js/components/Clock/clock.stories.js [hunk 1]
```diff
@@ -20,6 +45,15 @@ const AnalogClock = () => (
...
+      <Clock type="analog" />
+    </Box>
+  </Grommet>
+);
+
 storiesOf('Clock', module)
   .add('Digital Clock', () => <DigitalClock />)
-  .add('Analog Clock', () => <AnalogClock />);
+  .add('Analog Clock', () => <AnalogClock />)
+  .add('Custom Analog Clock', () => <CustomAnalogClock />);
```
Comment (ShimiSun) [inline src/js/components/Clock/clock.stories.js]: @britt6612 can you please remove all the `Clock` refs from the name of the stories?

File: src/js/components/Clock/doc.js [hunk 2]
```diff
@@ -52,3 +52,76 @@ export const doc = Clock => {
...
+
+export const themeDoc = {
+  'clock.analog.extend': {
+    description: 'Any additional style for Clock.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'clock.analog.hour.color': {
+    description: 'The color of the hour hand.',
+    type: 'string',
```
Comment (ShimiSun) [inline src/js/components/Clock/doc.js]: let's make sure `color` prop has the dark/light object as well on docs
`type: 'string | { dark: string, light: string }',`

File: src/js/components/Clock/doc.js [hunk 3]
```diff
@@ -52,3 +52,76 @@ export const doc = Clock => {
...
+    defaultValue: '24px',
+  },
+  'clock.analog.hour.width': {
+    description: 'The thickness of the hour hand',
+    type: 'string',
+    defaultValue: '8px',
+  },
+  'clock.analog.minute.color': {
+    description: 'The border to use in the accordion.',
```
Comment (ShimiSun) [inline src/js/components/Clock/doc.js]: wrong reference to `accordion`

File: src/js/components/Clock/doc.js [hunk 4]
```diff
@@ -52,3 +52,76 @@ export const doc = Clock => {
...
+    type: 'string',
+    defaultValue: '4px',
+  },
+  'clock.analog.second.color': {
+    description: 'The color of the seconds hand',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: '{ dark: accent-1, light: accent-1 }',
+  },
+  'clock.analog.second.shape': {
+    description: 'The border to use in the accordion.',
```
Comment (ShimiSun) [inline src/js/components/Clock/doc.js]: wrong reference to `accordion`

File: src/js/components/Clock/clock.stories.js [hunk 5]
```diff
@@ -20,6 +45,15 @@ const AnalogClock = () => (
...
+    <Box align="center" justify="start" pad="large">
+      <Clock type="analog" />
+    </Box>
+  </Grommet>
+);
+
 storiesOf('Clock', module)
-  .add('Digital Clock', () => <DigitalClock />)
-  .add('Analog Clock', () => <AnalogClock />);
+  .add('Digital', () => <DigitalClock />)
```
Comment (ShimiSun) [inline src/js/components/Clock/clock.stories.js]: Thanks for renaming.

Comment (britt6612) [reply src/js/components/Clock/clock.stories.js]: Welcome everything should be good now!

**Pull request merged**